firefox_puppeteer/ui/menu.py

```diff
@@ -37,6 +37,22 @@ def get_menu(self, label):
 
         return menu[0]
 
+    def get_menu_by_id(self, menu_id):
+        """Get a :class:`MenuElement` instance corresponding to the specified
+        ID.
+
+        :param menu_id: The ID of the menu, e.g., **file-menu** or
+        **view-menu**.
+        :returns: A :class:`MenuElement` instance.
+        """
+        menu = [m for m in self.menus if m.get_attribute('id') == menu_id]
```

Comment:
So as of now the current menu.py can only handle the mainmenu. It's because we rely on self.menus here. We have to make it more flexible similar to the link to Mozmill which I gave lately. We have to request the top element of any menu as best outside of the Menu class and pass it into the constructor. Lets base the Menu class also on UIBaseLib so that we have a reference back to the window.

Menus we should test are `main-menubar` and `contentAreaContextMenu`.